app/mailers/user_mailer.rb

```diff
@@ -391,7 +391,8 @@ def delete_signup_notification(user, challenge_signup)
     I18n.with_locale(Locale.find(@user.preference.preferred_locale).iso) do
       mail(
         to: user.email,
-        subject: "[#{ArchiveConfig.APP_SHORT_NAME}] Your sign-up for #{@signup.collection.title} has been deleted"
+        subject: "[#{ArchiveConfig.APP_SHORT_NAME}] Your sign-up for #{@signup.collection.title} has been deleted",
+        body: ""
```

Comment:
Yes - it needs a body. It looks like it's sent in the `before_destroy` callback in ChallengeSignup so I'm not sure what you mean about it not being used. ?